### PR TITLE
[ADD] Attendances: moving check in section to new doc

### PR DESCRIPTION
**Pull request by larm-odoo**
Moving the check in section of the main Attendances doc ot a nw doc, to shorten the main doc and cut up the sections into more bite-sized docs. Also updated the Access Rights section of the main Attendances doc to be more accurate and clear.

[Task card](https://www.odoo.com/odoo/3835/tasks/3941511?cids=3) for this PR.

### CODE REVIEW

**Comment by larm-odoo** [comment]
Hi @hojo-odoo - this is ready for a first round review! What I did was moved the Check In section to a new doc, and I also updated the Access Rights section of the main Attendances doc since it wasn't 100% currect or clear. That's what was changed for this PR.  Thank you!

File: content/applications/hr/attendances.rst
Comment (hojo-odoo) [inline content/applications/hr/attendances.rst:25]:
```suggestion
It is important to understand how the different access rights affect which features users can access in the
*Attendances* application.
```

File: content/applications/hr/attendances/check_in_check_out.rst
Comment (hojo-odoo) [inline content/applications/hr/attendances/check_in_check_out.rst:43]:
```suggestion
appears in that field, in an :guilabel:`HH:MM` (hours:minutes) format.
```

File: content/applications/hr/attendances/check_in_check_out.rst
Comment (hojo-odoo) [inline content/applications/hr/attendances/check_in_check_out.rst:58]:
```suggestion
of the widget, with the time the user checked in populating the time field. Beneath that line,
the hours and minutes that have elapsed since checking
in are displayed, in a :guilabel:`HH:MM`  format. As time passes, this value is updated to reflect the hours and minutes that have passed since
```

File: content/applications/hr/attendances/check_in_check_out.rst
Comment (hojo-odoo) [inline content/applications/hr/attendances/check_in_check_out.rst:63]:
```suggestion
HH:MM (AM/PM)` field appears in addition to the :guilabel:`Since HH:MM (AM/PM)` field. The times
displayed in each of these fields are the most recent check in time, and will match.
```

File: content/applications/hr/attendances/check_in_check_out.rst
Comment (hojo-odoo) [inline content/applications/hr/attendances/check_in_check_out.rst:64]:
```suggestion
Beneath the :guilabel:`Before HH:MM (AM/PM)` field, the previously logged hours are displayed, in an
```

File: content/applications/hr/attendances/check_in_check_out.rst
Comment (hojo-odoo) [inline content/applications/hr/attendances/check_in_check_out.rst:74]:
```suggestion
while the yellow :guilabel:`Check out` :icon:`fa-sign-out` button changes to a green :guilabel:`Check
```

**Review by hojo-odoo — APPROVED** [review]
Hey @larm-odoo,

Nice job on this doc. Feel free to move it to the next stage, once you take a look at my comments.

Thanks,
hojo

**Comment by larm-odoo** [comment]
Hi @ksc-odoo - this is ready for a final review!

File: content/applications/hr/attendances/check_in_check_out.rst
Comment (ksc-odoo) [inline content/applications/hr/attendances/check_in_check_out.rst:62]:
```suggestion
HH:MM (AM/PM)` field appears, in addition to the :guilabel:`Since HH:MM (AM/PM)` field. The time
```

File: content/applications/hr/attendances/check_in_check_out.rst
Comment (ksc-odoo) [inline content/applications/hr/attendances/check_in_check_out.rst:63]:
```suggestion
displayed in both of these fields match, and are populated with the most recent check in time.
```

File: content/applications/hr/attendances/check_in_check_out.rst
Comment (ksc-odoo) [inline content/applications/hr/attendances/check_in_check_out.rst:64]:
```suggestion
Beneath the :guilabel:`Before HH:MM (AM/PM)` field, the previously logged time is displayed, in an
```

File: content/applications/hr/attendances/check_in_check_out.rst
Comment (ksc-odoo) [inline content/applications/hr/attendances/check_in_check_out.rst:69]:
```suggestion
is the total time that is logged for the user, if they were to log out at that moment.
```

File: content/applications/hr/attendances/check_in_check_out.rst
Comment (ksc-odoo) [inline content/applications/hr/attendances/check_in_check_out.rst:88]: would you mind adding a bit more specificity to which dashboard? The main Odoo Dashboard? The main Attendances dashboard? Both? A little elaboration could be super helpful here. Thanks! 🙏

**Review by ksc-odoo — APPROVED** [review]
Hey, @larm-odoo -- just finished my Final Review. Great work. Approving now. Only have a handful of minor comments that require your attention. Once you implement the necessary changes/adjustments, feel free to tag this for Tech Review. Thanks! 👍

**Comment by larm-odoo** [comment]
Hi @samueljlieber - this is reay for a tech review! Just an FYI, when I run "make review" for the new file, it says Ineed to move the "is" to line 13- but if I do, it is over, so that doesn't seem right. Not sure if you'll encounter taht as well or if ti was an odd error for me.

File: content/applications/hr/attendances/check_in_check_out.rst
Comment (samueljlieber) [inline content/applications/hr/attendances/check_in_check_out.rst:2]: Remove the meta tag from `check_in_check_out.rst`
```suggestion
```

**Review by samueljlieber — APPROVED** [review]
Hi @larm-odoo! Thanks for your work on this PR to move the Clock in and out content to it's own doc. Your changes look good to me, with the exception of a couple comments I have bellow. Please make these corrections before merge :) Approving and delegating!

Please move the toctree to the bottom of the doc and add a seealso:: to `attendences.rst` to link to the sub-docs :)
```rst
.. seealso::
   - :doc:`attendances/check_in_check_out`
   - :doc:`attendances/hardware`

.. toctree::
   :titlesonly:

   attendances/check_in_check_out
   attendances/hardware
```
.....
@robodoo delegate=larm-odoo

**Pull request closed without merging**
